src/qml/WebPatchPage.qml

```diff
@@ -340,16 +340,17 @@ Page {
                     width: parent.width
                     contentHeight: filesContent.height
                     property bool isInstalled: !!container.versions && container.versions[modelData.project] == modelData.version
-                    property bool isCompatible: modelData.compatible.indexOf(PatchManager.osVersion) >= 0
-                    property bool isReinstallable: isInstalled && isCompatible
+                    property bool isCompatible: (modelData.compatible.indexOf(PatchManager.osVersion) >= 0)
+                    property bool forceCompatible: PatchManager.sfosVersionCheck !== VersionCheck.Strict
+                    property bool isReinstallable: isInstalled && (isCompatible || forceCompatible)
 
                     onClicked: {
-                        if ((PatchManager.sfosVersionCheck !== VersionCheck.Strict) && !isCompatible) {
+                        if (fileDelegate.isReinstallable) {
```

Comment:
consistency nit: is `fileDelegate.` needed
-  here specifically
- in all accesses for the properties changed in this PR (`isCompatible`, `forceCompatible`) or
-  in none of the cases?